addons/mail/models/mail_message.py

```diff
@@ -176,6 +177,13 @@ def default_get(self, fields):
     mail_ids = fields.One2many('mail.mail', 'mail_message_id', string='Mails', groups="base.group_system")
     canned_response_ids = fields.One2many('mail.shortcode', 'message_ids', string="Canned Responses", store=False)
 
+    def _compute_is_current_user_or_guest_author(self):
```

Comment:
missing dependencies, I see at least: context guest, context uid, author, author_guest_id (and technically current user groups but I think that one is fine to omit)